projects/ngx-datatable/src/lib/components/datatable.component.ts

```diff
@@ -85,6 +83,12 @@ import {
 export class DatatableComponent<TRow = any>
   implements OnInit, DoCheck, AfterViewInit, AfterContentInit, OnDestroy
 {
+  private scrollbarHelper = inject(ScrollbarHelper, { skipSelf: true });
+  private dimensionsHelper = inject(DimensionsHelper, { skipSelf: true });
```

Comment:
```suggestion
  private scrollbarHelper = inject(ScrollbarHelper);
  private dimensionsHelper = inject(DimensionsHelper);
```

Both are provided in the module. So for me it looks like skipSelf is pointless and we can just remove it